src/utils.js

```diff
@@ -65,3 +66,15 @@ export const npm = {
     }[npm.manager]);
   }
 };
+
+export async function installCodeshift(language) {
```

Comment:
So what if, similar to the `npm` util, we export a `codeshift` namespace with this `install` method? Then we can also define the codeshift binaries here as well. Thoughts?